### PR TITLE
Add CIFS backupstore

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn/longhorn#3599

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

File: content/docs/1.5.0/snapshots-and-backups/backup-and-restore/set-backup-target.md [hunk 1]
```diff
@@ -248,3 +249,51 @@ nfs://longhorn-test-nfs-svc.default:/opt/backupstore
...
 
 **Result:** Longhorn can store backups in NFS. To create a backup, see [this section.](../create-a-backup)
+
+### SMB/CIFS Backupstore
+
+Before configuring the SMB/CIFS backupstore, a credential secret for the backupstore can be created and deployed by
+  ```
+  #!/bin/bash
+
+  kubectl -n longhorn-system delete secret cifs-secret
```
Comment (innobead) [inline content/docs/1.5.0/snapshots-and-backups/backup-and-restore/set-backup-target.md:259]: Remove this cleanup statement.

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM. Just one comment needed to resolve.

File: content/docs/1.5.0/terminology.md [hunk 2]
```diff
@@ -9,11 +9,12 @@ weight: 4
 - [Backup target](#backup-target)
 - [Backup volume](#backup-volume)
 - [Block storage](#block-storage)
+- [CRD](#crd)
```
Comment (innobead) [inline content/docs/1.5.0/terminology.md:12]: This is another good example again to do the refactoring or a minor fix together with a new change. 

cc @longhorn/dev

File: content/docs/1.5.0/snapshots-and-backups/backup-and-restore/set-backup-target.md (hunk 1 above)
Comment (derekbit) [reply content/docs/1.5.0/snapshots-and-backups/backup-and-restore/set-backup-target.md]: Updated. Thank you.

**Review by innobead — APPROVED** [review]

**Pull request merged**